TLS/src/main/java/de/rub/nds/tlsattacker/dtls/protocol/handshake/ClientHelloDtlsMessage.java

```diff
@@ -22,10 +22,10 @@
 public class ClientHelloDtlsMessage extends ClientHelloMessage {
 
     @ModifiableVariableProperty(type = ModifiableVariableProperty.Type.COOKIE)
-    ModifiableByteArray cookie;
+    ModifiableByteArray cookie = null;
```

Comment:
What is the difference here?
Instance variables are by default initialize with `null`, correct?